lib/msal-common/src/config/Configuration.ts

```diff
@@ -72,18 +76,26 @@ export type LoggerOptions = {
     logLevel?: LogLevel
 };
 
+/**
+ * Telemetry info about library
+ */
+export type ClientInfo = {
```

Comment:
Client_Info is used for homeAccountId generation as we get `client_info` as a param in ServerResponse. I would recommend changing this name.

Reply:
Updated to LibraryInfo